test-suite/Makefile

```diff
@@ -258,7 +258,7 @@ $(addsuffix .log,$(wildcard bugs/closed/*.v)): %.v.log: %.v
 	    echo "    $<...Ok"; \
 	  else \
 	    echo $(log_failure); \
-	    echo "    $<...Error! (bug seems to be opened, please check)"; \
+	    echo "    $<...Error! (bug seems to be closed, please check)"; \
```

Comment:
Idem. If the bug was closed, we want to know when it is (inadvertently) reopened.